libs/common-services/src/test/java/uk/gov/di/ipv/core/library/helpers/TestVc.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.di.ipv.core.library.helpers;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.jackson.Jacksonized;
+import uk.gov.di.ipv.core.library.domain.BirthDate;
+import uk.gov.di.ipv.core.library.domain.NameParts;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.IDENTITY_CHECK_CREDENTIAL_TYPE;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.IDENTITY_CHECK_EVIDENCE_TYPE;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_FAMILY_NAME;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_GIVEN_NAME;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VC_NAME_PARTS;
+import static uk.gov.di.ipv.core.library.domain.VerifiableCredentialConstants.VERIFIABLE_CREDENTIAL_TYPE;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@Jacksonized
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TestVc {
+    @Builder.Default
+    private String[] type = {VERIFIABLE_CREDENTIAL_TYPE, IDENTITY_CHECK_CREDENTIAL_TYPE};
+
+    @Builder.Default
+    private TestCredentialSubject credentialSubject = TestCredentialSubject.builder().build();
+
+    @Builder.Default private List<TestEvidence> evidence = List.of(TestEvidence.builder().build());
+
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @Data
+    @Builder
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class TestCredentialSubject {
+        @Builder.Default
+        private List<Map<String, List<NameParts>>> name =
+                List.of(
+                        Map.of(
+                                VC_NAME_PARTS,
+                                List.of(
+                                        new NameParts("KENNETH", VC_GIVEN_NAME),
+                                        new NameParts("DECERQUEIRA", VC_FAMILY_NAME))));
+
+        @Builder.Default private List<BirthDate> birthDate = List.of(new BirthDate("1965-07-08"));
+
+        @Builder.Default
+        private List<Object> passport =
+                List.of(
+                        Map.of(
+                                "documentNumber", "321654987",
+                                "icaoIssuerCode", "GBR",
+                                "expiryDate", "2030-01-01"));
+
+        private List<Object> address;
+        private List<Object> socialSecurityRecord;
+        private List<Object> drivingPermit;
+        private List<Object> residencePermit;
+        private List<Object> idCard;
```

Comment:
I was wondering if these should all have their objects modeled as classes. That way it's easier to enforce that the correct attributes are being used.

Reply:
I think that's more the work of Mike's ticket https://govukverify.atlassian.net/browse/PYIC-5032

Reply:
Yeah that makes sense. Would be good to come back and update these fixtures after his is in